.github/workflows/ci.yml

```diff
@@ -49,14 +45,13 @@ jobs:
       - name: Run vimeo/psalm on internal code
         run: ./vendor/bin/psalm --config=psalm.xml --no-progress --shepherd --show-info=false --stats
 
-  tests:
+  compiler-tests:
+    name: Compiler tests on PHP ${{ matrix.php }}
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        operating-system: [ 'ubuntu-latest', 'windows-latest', 'macos-latest' ]
         php: [ '7.4', '8.0' ]
```

Comment:
Should we add also PHP 8.1 or let's wait until we migrate completely to PHP 8.0?

Reply:
Good idea, but I would first move to 8.0, and then you can add a follow up PR adding the 8.1, because we might have to fix some broken things (luckily not). But just in case, I would delay that for another iteration.

Reply:
Not all our dependencies are 8.1 ready yet, I guess. So we can do that later in another PR